release_notes/40-release-notes.adoc

```diff
@@ -455,6 +456,21 @@ Release date: 18 May 2023
 * Fixed issue with manifest-based secured cluster installations for {rh-rhacscs-first}.
 * Fixed issue with unwanted logging for {product-title-managed-short}.
 
+[id="resolved-in-version-402"]
+=== Resolved in version 4.0.2
+
+Release date: 31 May 2023
+
+//bugs are not linked because ACS Jira is not public
+* This release of {product-title-short} includes a fix for:
+
+** link:https://access.redhat.com/security/cve/cve-2023-24540[CVE-2023-24540], by building {product-title-short} with updated Golang.
+//ROX-16989
+** link:https://access.redhat.com/security/cve/CVE-2023-29400[CVE-2023-29400], which fixes a vulnerability in Golang that affects templates with actions in unquoted HTML attributes.
+** link:https://access.redhat.com/security/cve/CVE-2023-24539[CVE-2023-24539], which fixes a vulnerability in Golang related to the handling of angle brackets in Cascading Style Sheets (CSS) contexts.
+
+//ROX-17189
+* Fixed an issue that caused frequent Central pod restarts with the `context deadline exceeded` error.
```

Comment:
Does `context deadline exceeded` have `C` in caps? If not, this is good.

Reply:
"c" is in lower case